Src/Extern/Conduit/AMReX_Conduit_Blueprint_ParticlesI.H

```diff
@@ -94,65 +117,79 @@ ParticleTileToBlueprint(const ParticleTile<ParticleType,
     // -----------------------------
 
     int vname_real_idx = 0;
-    // struct real fields, the first set are always the particle positions
-    // which we wrap above
-    for (int i = 0; i < NStructReal; i++)
+    if constexpr(!ParticleType::is_soa_particle)
```

Comment:
we need to build the fields for the SOA case, not sure how to access from AMReX but it should be a an else case.